quirks/password-rules.json

```diff
@@ -548,6 +548,9 @@
     "telekom-dienste.de": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; required: [#$%&()*+,./<=>?@_{|}~];"
     },
+    "thameswater.co.uk": {
+        "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; required: [#$%&()*+,./<=>?@_{|}~];"
```

Comment:
Did you experiment to come up with his specific set of special characters? Or did you copy it from somewhere else? Could we use `required: special`?

Reply:
Sorry yes just copy / pasted. Happy to change to simplify it.